packages/core/src/test/video.test.tsx

```diff
@@ -56,6 +56,18 @@ describe('Render correctly with props', () => {
 			)
 		).not.toThrow();
 	});
+	test('It should render Video with children source', () => {
```

Comment:
From what I've tested, the implementation in this PR still requires the user to pass the `src` prop.

If I understood correctly, we want the user to choose between the `src` and the `children` props as the native tags, right?

Reply:
Yes I agree with what @arthurdenner said. Still needs some work, if necessary we can support you!